server/src/main/java/org/opensearch/search/aggregations/bucket/histogram/DateHistogramAggregator.java

```diff
@@ -171,6 +191,13 @@ public LeafBucketCollector getLeafCollector(LeafReaderContext ctx, LeafBucketCol
         if (optimized) throw new CollectionTerminatedException();
 
         SortedNumericDocValues values = valuesSource.longValues(ctx);
+        CompositeIndexFieldInfo supportedStarTree = getSupportedStarTree(this.context);
```

Comment:
can we benchmark how date histogram aggs with just doc count fares against the point query optimizations ? [ since we skip using star tree in such cases , it'll be better to know if there are scenarios where we can use star tree as well ]

we can add a sub issue for post 2.19 for this

Reply:
Sure, will add up a follow-up task to benchmark this in [Search Plan](https://github.com/opensearch-project/OpenSearch/issues/15257).